### PR TITLE
feature/issue-534

### PR DESCRIPTION
**Pull request by GiuntaLucas**
Display the import default list when no tokens are already imported.

### CODE REVIEW

File: src/components/TokenList/TokenList.tsx [hunk 1]
```diff
@@ -184,7 +184,6 @@ const TokenList = ({
               </TokenContainer>
             )}
             {inactiveTokens.length !== 0 &&
-              tokenQuery &&
               sortedFilteredTokens.length < 5 && (
```
Comment (piersss) [inline src/components/TokenList/TokenList.tsx:187]:
```suggestion

```

This line should also be removed or else the tokens still won't appear if the `sortedFilteredTokens` are more than 5. All else seems fine!

**Review by piersss — CHANGES_REQUESTED** [review]

File: src/components/TokenList/TokenList.tsx (hunk 1 above)
Comment (GiuntaLucas) [reply src/components/TokenList/TokenList.tsx]: I removed this condition but i have another question: Do we still show to the user only 10 token from the inactive one by default or do we have to increase the list ? Maybe we can increase to 50 - 100, what do you think ? 

image

Comment (piersss) [reply src/components/TokenList/TokenList.tsx]: Not sure, did you ask around in the dev chat yet?

**Review by piersss — APPROVED** [review]

File: src/components/TokenList/TokenList.tsx [hunk 2]
```diff
@@ -107,10 +107,8 @@ const TokenList = ({
 
   // only take the top 10 tokens
   const inactiveTokens: TokenInfo[] = useMemo(() => {
-    return filterTokens(Object.values(sortedInactiveTokens), tokenQuery!).slice(
-      0,
-      10
-    );
+    const tokens = filterTokens(Object.values(sortedInactiveTokens), tokenQuery!);
+    return tokens.length >= 100 ? tokens.slice(0, 100) : tokens;
```
Comment (piersss) [inline src/components/TokenList/TokenList.tsx:111]: `return filterTokens(Object.values(sortedInactiveTokens), tokenQuery!).slice(
      0,
      10
    );
`

**Pull request merged**